.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: go
+
+go:
+  - "1.11.x"
+  - master
+
+env:
+  - DEP_VERSION=0.5.0
+  - NODE_TEST_IMAGE_VERSION=10.15.0
+
+services:
+  - docker
+
+before_install:
+  - curl -L -s https://github.com/golang/dep/releases/download/v${DEP_VERSION}/dep-linux-amd64 -o $GOPATH/bin/dep
+  - chmod +x $GOPATH/bin/dep
+
+  - docker pull node:${NODE_TEST_IMAGE_VERSION}
```

Comment:
Why do we have to pull the Docker images? Dunner should pull the image itself, right?

Reply:
Oh, got it!